relay-server/src/services/processor.rs

```diff
@@ -1606,6 +1601,14 @@ impl EnvelopeProcessorService {
             )?;
         });
 
+        if_processing!(self.inner.config, {
+            ourlog::extract_from_event(
+                managed_envelope,
+                &event,
+                &self.inner.global_config.current(),
+            );
+        });
```

Comment:
I think we should move this to line 1614 (after `event::scrub`) so we get PII scrubbing for free.